tests/dashboards/utils.py

```diff
@@ -0,0 +1,37 @@
+from superset import security_manager
+
+
```

Comment:
this should be DashboardTestUtilsClass where you not only have all existing methods you also manage local state of created dashboards and dashboardRoles and permissions so you can clean after every test/alltests